test/mlir/onnx/onnx_recompose_combine_parallel_dense.mlir

```diff
@@ -0,0 +1,101 @@
+// RUN: onnx-mlir  --useOnnxModelTypes=false --EmitONNXIR --printIR %s | FileCheck %s
+
+func.func @test_gemm_concat_simple(%arg0: tensor<1x4xf32>) -> tensor<1x6xf32> {
+  %0 = onnx.Constant dense<[[6.033820e-01, 0.874853491, 0.840596497], 
```

Comment:
Since we do not check the output accuracy, it's better to use splat constants to make the program clean, e.g. `%0 = onnx.Constant dense<5.5>: tensor<4x3xf32>`